nosqlbench/http-jsonapi-crud-dataset.yaml

```diff
@@ -81,47 +85,94 @@ blocks:
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>/<<collection:docs_collection>>
         Accept: "application/json"
         X-Cassandra-Request-Id: "{request_id}"
         X-Cassandra-Token: "{token}"
         Content-Type: "application/json"
         ok-body: ".*\"insertedIds\":\"{seq_key}\".*"
-        body: "{document_json}"
-
+        body: >2
+          {
+            "insertOne" : {
+              "document" : { 
+                "_id" :         "{seq_key}",
+                "data" :      {document_json_without_id}
+              }
+            }
+          }
+          
+  read:
+    ops:
+      # aka findOne with _id as filter
       read-document:
         space: "{space}"
-        method: GET
-        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v2/namespaces/<<namespace:jsonapi_crud_dataset>>/collections/<<table:docs_collection>>/{random_key}
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>/<<collection:docs_collection>>
         Accept: "application/json"
         X-Cassandra-Request-Id: "{request_id}"
         X-Cassandra-Token: "{token}"
         Content-Type: "application/json"
         ok-body: ".*\"data\".*"
+        body: >2
+          {
+            "findOne" : {
+              "filter" : {
+                "_id" : "{random_key}"
+              }
+            }
+          }  
 
+  update:
+    ops:
+      # aka updateOne
+      # for parity with other tests this only uses set, not unset, no return value
       update-document:
         space: "{space}"
-        method: PUT
-        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v2/namespaces/<<namespace:jsonapi_crud_dataset>>/collections/<<table:docs_collection>>/{random_key}
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>/<<collection:docs_collection>>
         Accept: "application/json"
         X-Cassandra-Request-Id: "{request_id}"
         X-Cassandra-Token: "{token}"
         Content-Type: "application/json"
-        body: "{document_json}"
+        # because this is not an upsert, modified count could technically be 0 or 1,
+        # but since we are fixing the _ids to be sequential over the docscount range during the write phase,
+        # every update during this phase should update exactly 1 document.
+        ok-body: ".*\"modifiedCount\":[0,1].*"
+        body: >2
+          {
+            "updateOne" : {
+              "filter": {
+                "_id" : "{random_key}"
+              },
+              "update": {
+                "$set": {
+                  "update_field": "update_value"
```

Comment:
let's randomize this, not sure if the set will be performed if this is already the value:

```
"update": {
                "$set": {
                  "update_field": "{seq_key}"
```